src/cgame/etj_jump_speeds.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 ETJump team <zero@etjump.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#pragma once
+
+#include "cg_local.h"
+
+namespace ETJump
+{
+	constexpr int MAX_JUMPS = 10;
+	std::vector<int> jumpSpeedHistory;		// last 10 jump speeds
+	bool jumpSpeedDeleted;
+	bool resetQueued;
+	int lastDeletedSpeed;					// last jump speed that was deleted from history
+	int team;
+
+	void DrawJumpSpeeds();
+	void UpdateJumpSpeeds();
+	void QueueJumpSpeedsReset();
```

Comment:
What's the use case for this vs. `ResetJumpSpeeds`?

Reply:
Since the drawing function runs every client frame, I made a queue system for resetting, this allows you to see your previous jump speeds after load/death, and the view is only cleared once you jump again. If I would just call a reset instantly, you would not be able to see your jump speeds after load.